tests/functional/adapter/test_basic.py

```diff
@@ -0,0 +1,152 @@
+import pytest
```

Comment:
nice testing, but agina as here: https://github.com/dbt-athena/dbt-athena/pull/202/files#r1164195147 please keep minimal changes, makes the review easier :)